persistence/persistence-api/src/main/java/org/cbioportal/persistence/CacheDisableSetting.java

```diff
@@ -0,0 +1,40 @@
+package org.cbioportal.persistence;
```

Comment:
instead of naming this class `CacheDisableSetting` might I suggest a different name like `DisabledCacheConfig` or similar?

Reply:
sounds good .. renaming

Reply:
actually, using CacheEnabledConfig .. to match the portal.properties setting "ehcache.cache_enabled"

Reply:
sounds good to me!